README.md

```diff
@@ -30,4 +28,10 @@ Add Body sample:
 {
     "content":"fried noodles"
 }
-```
\ No newline at end of file
+```
+
+## To test Rojak Search manually
+`from Rojak import Rojak`
+`rojak = Rojak()`
+`rojak.search("spicy curry chicken with vegetable")`
```

Comment:
```py
    from Rojak import Rojak
    rojak = Rojak()
    rojak.search("spicy curry chicken with vegetable")     
    ```
    
This will look nicer and provides syntax highlighting. You should probably also use this same style for detailing the info in the PR.

Reply:
Done!